### ISSUE TITLE
Error while installing smudgeplot

### ISSUE THREAD

**Issue by NWybouw** [tbenavi1/smudgeplot#16]
Dear, 

I'm getting a strange ERROR message while installing smudgeplot that I cannot directly place.
Do I perhaps have an incorrect version of devtools?

```
You are a coding rockstar!
Warning message:
`encoding` is deprecated; all files now assumed to be UTF-8 
Installing package into ‘/home/nicky/R/x86_64-pc-linux-gnu-library/3.4’
(as ‘lib’ is unspecified)
Warning: invalid package ‘../smudgeplot’
Error: ERROR: no packages specified
Warning message:
In install.packages("../smudgeplot", repos = NULL, type = "source") :
  installation of package ‘../smudgeplot’ had non-zero exit status
```

**Comment by KamilSJaron** [comment]
I could not reproduce the error.

Do you have any progress by any chance?

**Comment by NWybouw** [comment]
Dear Kamil,


Thank you for your response. I ran the installation in R and this is the output:


> library(devtools)
>
> document()
Updating smudgeplot documentation
Loading smudgeplot
Loading required package: viridis
Loading required package: viridisLite
Loading required package: argparse
Writing NAMESPACE
Writing NAMESPACE
Warning message:
roxygen2 requires Encoding: UTF-8
> test()
Loading smudgeplot
Loading required package: testthat

Attaching package: ‘testthat’

The following object is masked from ‘package:devtools’:

    setup

Testing smudgeplot
✔ | OK F W S | Context
✔ |  1       | 1d subsets fitting
✔ |  1       | integration
✔ |  2       | smudge container generation
✔ |  4       | generation of peak summary [0.3 s]

══ Results ════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════
Duration: 0.5 s

OK:       8
Failed:   0
Warnings: 0
Skipped:  0
Warning message:
`encoding` is deprecated; all files now assumed to be UTF-8
> install.packages("../smudgeplot", repos = NULL, type="source")
Installing package into ‘/home/nicky/R/x86_64-pc-linux-gnu-library/3.4’
(as ‘lib’ is unspecified)
Warning: invalid package ‘../smudgeplot’
Error: ERROR: no packages specified
Warning in install.packages :
  installation of package ‘../smudgeplot’ had non-zero exit status



The dir() is:


> dir()
 [1] "DESCRIPTION"       "FAQ.md"            "hetkmers.py"       "install.R"         "kmer_cov_cutoff.R" "Makefile"
 [7] "man"               "NAMESPACE"         "playground"        "R"                 "README.md"         "smudgeplot.R"
[13] "tests"



And the getwd() is:


> getwd()
[1] "/home/nicky/sibling_kmers"




Thank you for your help!

all the best,

Nicky




________________________________
Van: Kamil S. Jaron <notifications@github.com>
Verzonden: dinsdag 25 september 2018 17:29
Aan: tbenavi1/smudgeplot
CC: NWybouw; Mention
Onderwerp: Re: [tbenavi1/smudgeplot] Error while installing smudgeplot (#16)


Hello @NWybouw<https://github.com/NWybouw> that's indeed strange.

Warning: invalid package ‘../smudgeplot’ sounds like the directory with smudgeplot is not there. Have you renamed the directory by any chance? Could you perhaps write in R these two commands and write me the output: dir() and getwd()?

**Comment by KamilSJaron** [comment]
Hello Nicky,

this is our fault. The original name of the repository was "sibling_kmers", you need to rename your directory to smudgeplot to make it work or even easier, remove directory `sibling_kmers`, redownload, reinstall it.

In `bash` you can type this

```
cd /home/nicky/
rm -rf sibling_kmers
git clone https://github.com/tbenavi1/smudgeplot
cd smudgeplot
Rscript install.R
```

if it still does not work, let me know (I will reopen the issue in that case).

**Comment by KamilSJaron** [comment]
I realized that I wrote stupidly the installation script. In 7795c188f750b410df534fdd910f838a3836464b it's fixed, now it should install whatever the name of directory is.

**Comment by KamilSJaron** [comment]
HA, now I found where is the REAL issue. It's OUR documentation (thanks Acer for pointing it out).

README fixed in 4f60ef0d6fcb771d49aa8905789670cc34f1d68e
